_episodes/02-project-planning.md

```diff
@@ -1,110 +1,143 @@
+>
+> > ## Solution
+> > Errors:
+> > - Sequential order of well_position changes
+> > - Format of client_sample_id changes and cannot have spaces, slashes, non-standard ASCII characters
+> > - Capitalization of the replicate column changes
+> > - Volume and concentration column headers have unusual (not allowed) characters
+> > - Volume, concentration, and RIN column decimal accuracy changes
+> > - The prep_date and ship_date formats are different, prep_date has multiple formats 
+> > - Are there others not mentioned?
+> >
+> > Improvements in naming
+> > - Shorten client_sample_id names, and maybe just call them "names"
+> >   - For example: "wt" for "wild-type". Also, they are all "1hr" so that is superfluous information
+> > - The prep_date and ship_date might not be needed
+> > - Use "microliters" for "Volume (Â¬ÂµL)" etc. 
+> >
+> > Errors hard to spot:
+> > - No space between "wild" and "type", repeated barcode numbers, missing data, duplicate names
+> > - Find by sorting, or counting
+> >
+> {: .solution}
+{: .challenge}
+
+# Retrieving samples from the facility
+
+When the data come back from the sequencing facility, you will receive some documentation (metadata) as well as
+the sequence files themselves. Download and examine the following example file - here provided as a text file and
+Excel file:
+
+- [Sequencing results - text](../files/sequencing_results_metadata.txt)
+- [Sequencing results - Excel](../files/sequencing_results_metadata.xls)
+
+> ## Exercise
+>
+> 1. How are these samples organized?
+> 2. If you wanted to relate file names to the sample names submitted above (e.g. wild type...) could you do so?
+> 3. What do the \_R1/\_R2 extensions mean in the file names?
+> 4. What does the '.gz' extension on the filenames indicate?
+> 5. What is the total file size - what challenges in downloading and sharing these data might exist?  
+>
+> > ## Solution
+> >
+> > - Samples are organized by sample_id
+> > - To relate filenames use the sample_id, and do a VLOOKUP on submission sheet
+> > - The \_R1/\_R2 extensions mean "Read 1 and "Read 2 of each sample
+> > - The '.gz' extension means it is a compressed "gzip" type format to save disk space
+> > - The size of all the files combined is 1113.60 gb (over a terabyte!). To move files this large will
+> >   require using checksums to validate file integrity after transfers, and a fast transfer protocol
+> >   or software
+> >
+> {: .solution}
+{: .challenge}
+
+# Storing data
+
+The raw data you get back from the sequencing center is the foundation of your sequencing analysis. You need to keep this data, so that you can always come back to it if there are any questions or you need to re-run an analysis, or try a new analysis approach.
+
+### Guidelines for storing data
+
+- Store the data in a place that is accessible by you and other members of your lab. At a minimum, you and your PI should have access

```

Comment:
I'm not sure if "PI" is a term that is used outside of the US. Recommend changing to "you and your supervisor" or "you and the head of your lab".